lib/mods/replace-methods.js

```diff
@@ -8,6 +8,24 @@ const {
   getOriginal,
 } = require('../originals-helper.js');
 
+/**
+ *
+ * @param {Node} node - the node to examine
+ * @param {string} baseTypeName - the base type of the node
+ *
+ * @return {boolean} whether the method is a namespace method
+ */
+function isNamespaceMethod(node, baseTypeName) {
+  if (node.getExpression().getSymbol().getDeclarations()[0]
+      && node.getExpression().getSymbol().getDeclarations()[0]
```

Comment:
Go for [DRY code](https://www.softwareyoga.com/is-your-code-dry-or-wet/). Save this in a variable instead of repeating it three times.